.github/workflows/test.yml

```diff
@@ -6,50 +6,58 @@ on:
   pull_request:
 
 jobs:
-  test:
+  cases:
+    name: ${{ matrix.os }}, python${{ matrix.python-version }}, ${{ matrix.case-name }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         include:
-          - os: ubuntu-latest
+          - case-name: qutip@dev.major
+            os: ubuntu-latest
             qutip-version: '@dev.major'
             qiskit-version: ''
             pyqir-version: ''
             python-version: '3.10'
-          - os: windows-latest
-            qutip-version: '==4.6.*'
-            qiskit-version: ''
+          - case-name: qutip@master
+            os: ubuntu-latest
+            qutip-version: '@master'
```

Comment:
It's likely that we'll soon freeze qutip master and move development of 4.7.X to  `qutip-4.7.X` so that we can eventually use master for v5. You could switch to 4.7.X now, or just do it later when master switches over.

Reply:
Thank! Yes, that makes sense. All the tests should still pass in that case. I'll switch later.